web/pdf_find_bar.js

```diff
@@ -119,39 +119,46 @@ class PDFFindBar {
         status = "pending";
         break;
       case FindState.NOT_FOUND:
-        findMsg = this.l10n.get("pdfjs-find-not-found");
+        findMsgId = "pdfjs-find-not-found";
         status = "notFound";
         break;
       case FindState.WRAPPED:
-        findMsg = this.l10n.get(
-          `pdfjs-find-reached-${previous ? "top" : "bottom"}`
-        );
+        findMsgId = `pdfjs-find-reached-${previous ? "top" : "bottom"}`;
         break;
     }
-    this.findField.setAttribute("data-status", status);
-    this.findField.setAttribute("aria-invalid", state === FindState.NOT_FOUND);
+    findField.setAttribute("data-status", status);
+    findField.setAttribute("aria-invalid", state === FindState.NOT_FOUND);
 
-    findMsg.then(msg => {
-      this.findMsg.setAttribute("data-status", status);
-      this.findMsg.textContent = msg;
-    });
+    findMsg.setAttribute("data-status", status);
+    if (findMsgId) {
+      findMsg.setAttribute("data-l10n-id", findMsgId);
+    } else {
+      findMsg.removeAttribute("data-l10n-id");
+      findMsg.textContent = "";
```

Comment:
*Please note:* I don't know if the is the "best" method to clear a previously translated element, but if there's a nicer/shorter way of doing it I'm all ears.

Reply:
I don't see anything here:
https://searchfox.org/mozilla-central/source/dom/webidl/DOMLocalization.webidl
which could indicates a better way to do.